src/api/propensity-api.js

```diff
@@ -287,38 +110,28 @@ export let PropensityScore;
  *         enum listed in the Event enum above.
  *
  *  @typedef {{
- *    name: string,
+ *    name: !Event,
```

Comment:
this might be a problem since publishers dont have access to enum types. a good way to confirm is try using this api is scenic demo.